pnpm-lock.yaml

```diff
@@ -11954,31 +11958,42 @@ packages:
     resolution: {integrity: sha512-LYvhNKfwWSPpocw8GI7gpK2nq3HSDuEPC/uSYaALSJu9xjsalaaYFOq0Pwt5KmVqwEbZlDu81aLXwBOmD/Fv9g==}
     engines: {node: '>=6.9.0'}
     dependencies:
-      '@babel/highlight': 7.18.6
+      '@babel/highlight': 7.22.10
+
+  /@babel/code-frame/7.22.10:
+    resolution: {integrity: sha512-/KKIMG4UEL35WmI9OlvMhurwtytjvXoFcGNrOvyG9zIzA8YmPjVtIZUf7b05+TPO7G7/GEmLHDaoCgACHl9hhA==}
+    engines: {node: '>=6.9.0'}
+    dependencies:
+      '@babel/highlight': 7.22.10
+      chalk: 2.4.2
 
   /@babel/compat-data/7.21.7:
     resolution: {integrity: sha512-KYMqFYTaenzMK4yUtf4EW9wc4N9ef80FsbMtkwool5zpwl4YrT1SdWYSTRcT94KO4hannogdS+LxY7L+arP3gA==}
     engines: {node: '>=6.9.0'}
 
+  /@babel/compat-data/7.22.9:
```

Comment:
looks like we have both babel 7.21 and 7.22, which is causing a lot of new deps. Maybe we can upgrade babel to 7.22 instead of adding a second set of all its packages?

Reply:
The dependency on 7.21 comes via a transitive dependency of storybook.  Moving storybook forward churns pnpm-lock.yaml a lot (-544, +321) and makes the diff hard to read, but I do think consolidates the versions of babel.

It seems worth doing, but maybe as a separate change?